metron-deployment/other-examples/manual-install/Manual_Install_CentOS6.md

```diff
@@ -638,15 +638,15 @@ Edit crontab with `# crontab -e` and add:
 
 bro-kafka:
 ```
-# cp -r /root/metron/metron-sensors/bro-plugin-kafka /tmp
-# cd /tmp/bro-plugin-kafka
+# git clone https://github.com/apache/metron-bro-plugin-kafka /tmp/metron-bro-plugin-kafka
+# cd /tmp/metron-bro-plugin-kafka
 # rm -rf build/
```

Comment:
Why no make clean?

Reply:
Not a bad idea - I was just migrating the instructions to use the new location without changing anything else.